src/Pos/Delegation/DB.hs

```diff
@@ -132,6 +154,20 @@ instance DBIteratorClass DlgTransRevIter where
 getDelegators :: MonadDBRead m => Source (ResourceT m) (StakeholderId, HashSet StakeholderId)
 getDelegators = dbIterSource GStateDB (Proxy @DlgTransRevIter)
 
+-- Iterator over the "this epoch posted" set.
+data ThisEpochPostedIter
+
+instance DBIteratorClass ThisEpochPostedIter where
+    type IterKey ThisEpochPostedIter = StakeholderId
+    type IterValue ThisEpochPostedIter = ()
+    iterKeyPrefix = iterPostedThisEpochPrefix
+
+-- | Get all keys of thisEpochPosted set.
+getThisEpochPostedKeys :: MonadDBRead m => m [StakeholderId]
```

Comment:
It would be better to return `HashSet` I think. It's more convenient because I don't think someone may need this data in other form than a set. And its more optimal, because you can construct a `HashSet` without constructing intermediate list.
Perhaps it's not really important.